pyproject.toml

```diff
@@ -0,0 +1,109 @@
+[project]
+name = "hgvs"
+authors = [
+  { name="HGVS Contributors", email="biocommons-dev@googlegroups.com" },
+]
+description = "HGVS Parser, Formatter, Mapper, Validator"
+readme = "README.rst"
+license = { file="LICENSE.txt" }
+requires-python = ">=3.8"
```

Comment:
You might want to update the pypi docs
```
Important: biocommons packages require Python 3.6+. [More](https://groups.google.com/forum/#!topic/hgvs-discuss/iLUzjzoD-28)
```

Was there a similar discussion around bumping this to 3.8? We are using hgvs in some projects that are on 3.6

cc @reece @andreasprlic

Reply:
Any news on it @reece ?

Reply:
Without bumping the major version, this change can be breaking for anyone using hgvs. @reece would you mind reverting to 3.6? We dont see any apparent code that requires 3.8 features.